extensions/amp-timeago/0.2/timeago.js

```diff
@@ -14,39 +14,41 @@
  * limitations under the License.
  */
 
-import {useResourcesNotify} from '../../../src/preact/utils';
+import {createElement} from '../../../src/preact';
```

Comment:
Did you end up needing to fork preact?

Reply:
It's not a fork, just a wrapper around the APIs. Closure eliminates all of it when it compiles.